inspector/src/components/actionTabs/tabs/propertyGrids/materials/texturePropertyGridComponent.tsx

```diff
@@ -37,17 +37,24 @@ interface ITexturePropertyGridComponentProps {
     onPropertyChangedObservable?: Observable<PropertyChangedEvent>
 }
 
-export class TexturePropertyGridComponent extends React.Component<ITexturePropertyGridComponentProps> {
+interface ITexturePropertyGridComponentState {
```

Comment:
I migrated this to state because we want to trigger a re-render every time we open or close the texture editor popout